### ISSUE TITLE
demo toy problem: blank prediction output 

### ISSUE THREAD

**Issue by gopi77** [jakeret/tf_unet#97]
Hi
Tried running the demo_toy_problem.ipynb in jupyter notebook.
But the prediction output window is blank.
![image](https://user-images.githubusercontent.com/12639784/30413679-1fb36062-993d-11e7-96d8-2464cd9ebe32.png)

The training output (last 3 lines below)
2017-09-14 09:44:04,354 Epoch 9, Average loss: 0.3072, learning rate: 0.1260
2017-09-14 09:44:04,537 Verification error= 16.5%, loss= 0.5079
2017-09-14 09:44:07,550 Optimization Finished!

I got up to epoch_9.jpg
![epoch_9](https://user-images.githubusercontent.com/12639784/30413635-de04665c-993c-11e7-8203-293865f823d2.jpg)

1. Anyone faced this kind of problem ?
2. Also I would like to feed my own image and test it. Not sure about the API modification. Please help
    a. x_test, y_test = generator(1)
    b. prediction = net.predict("./unet_trained/model.cpkt", x_test)

My PC details:
OS: Ubuntu 16.04
tensorflow 1.2.1
Python 2.7.13 |Anaconda custom (64-bit)| (default, Dec 20 2016, 23:09:15) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux2

**Comment by jakeret** [comment]
Have you checked the values of the prediction? For the visualization I'm clipping at > 0.9. If the network is uncertain the probability will be lower hence the black image

**Comment by gopi77** [comment]
I tried clipping at > 0.7 & got output, thanks.
![image](https://user-images.githubusercontent.com/12639784/30426405-911fd10a-9969-11e7-8555-bafdc9e242f6.png)

**Comment by abderhasan** [comment]
Can you kindly explain where clipping would be performed?

Thanks.
